packages/customer/src/migrations/Migration20240124154000.ts

```diff
@@ -0,0 +1,79 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20240124154000 extends Migration {
+  async up(): Promise<void> {
+    // Adjust existing 'customer' table
+    this.addSql(
+      'alter table "customer" add column if not exists "company_name" text null;'
```

Comment:
Aren't we missing a create table if not exists here?

Reply:
if you use the module on a separate database, the customer won't exists

Reply:
Yes you are right! See latest fix that should work right?

Reply:
we are still missing the create table in the last commit

Reply:
the module can be started in a different database, in that case the user can use the runMigration function to spin up his table in his database